### ISSUE TITLE
Recommendation page

### ISSUE THREAD

**Issue by archanagandhi** [QAZIMAAZARSHAD/Movie-Streaming-Website#60]
Recommendation or mood page which will suggest movies according to the mood

**Comment by QAZIMAAZARSHAD** [comment]
Go ahead.

**Comment by nik464** [comment]
please assign me this . I will surely design recommendations page as LGM SOC participant.

**Comment by QAZIMAAZARSHAD** [comment]
@archanagandhi are you working on this?

**Comment by archanagandhi** [comment]
No, I am having sem exams. couldn't concentrate. Let me close the issue

**Comment by nitishapiplani** [comment]
if  u are closing may i do it

**Comment by kshitija-2001** [comment]
Please assign me this issue under LGMSOC 21

**Comment by utkarsh-ub** [comment]
@kshitija-2001 @nitishapiplani any updates?
